.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+### Summary
+<What's this change about?>
```

Comment:
You can leave this as an HTML comment, so that people don't need to delete them:

```suggestion
<!-- What's this change about? -->
```

Same for the others.